.github/workflows/generateAutoConfigs.yaml

```diff
@@ -110,7 +110,7 @@ jobs:
         run: |
           set -x
           # stashes the source branch with generated libs 
-          MESSAGE=`echo "feat(generated): Generate new autoconfig libraries"`
+          MESSAGE=`echo "chore(codegen): update starter modules in spring-cloud-preview"`
```

Comment:
Can you just keep it as "chore" instead of "chore(codegen):" ? You can probably add the codegen keyword after the colon.

Reply:
For sure - updated.